lib/drawers/health.vue

```diff
@@ -156,6 +165,16 @@
         </li>
       </ul>
     </div>
+    <div v-for="(warning, index) in kilnjsWarnings" class="publish-warning" :key="`publish-kilnjs-warning-item-${index}`">
+      <span class="warning-label">{{ warning.label }}</span>
+      <span class="warning-description">{{ warning.description }}</span>
+      <span class="validation-info">Go To Components</span>
+      <ul class="validation-items">
+        <li v-for="(item, idx) in warning.items" class="validation-item" :key="`publish-warning-validation-item-${idx}`">
+          <span class="validation-item-location" :class="{ 'validation-item-link': item.uri && item.field }" @click.stop="openLocation(item.uri, item.field, item.path, item.location)">{{ item.location }}</span> <span v-if="item.preview" class="validation-item-preview">{{ item.preview }}</span>
+        </li>
+      </ul>
+    </div>
```

Comment:
The error (lines 148-157) and warning (lines 168-177) elements are basically the same, do you think it would be possible to create a reusable component to display these elements?

Reply:
Not only possible, but also a great idea.  Done.